internal/pkg/admission/policy-server-ca-secret.go

```diff
@@ -128,16 +191,96 @@ func (r *Reconciler) buildPolicyServerCARootSecret(generateCA generateCAFunc, pe
 	}
 	caPrivateKeyBytes := x509.MarshalPKCS1PrivateKey(caRoot.CaPrivateKey)
 	secretContents := map[string][]byte{
-		constants.PolicyServerCARootCACert:             caRoot.CaCert,
-		constants.PolicyServerCARootPemName:            caPEMEncoded,
-		constants.PolicyServerCARootPrivateKeyCertName: caPrivateKeyBytes,
+		constants.KubewardenCARootCACert:             caRoot.CaCert,
+		constants.KubewardenCARootPemName:            caPEMEncoded,
+		constants.KubewardenCARootPrivateKeyCertName: caPrivateKeyBytes,
 	}
 	return &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      constants.PolicyServerCARootSecretName,
+			Name:      constants.KubewardenCARootSecretName,
 			Namespace: r.DeploymentsNamespace,
 		},
 		Data: secretContents,
 		Type: corev1.SecretTypeOpaque,
 	}, nil
 }
+
+// ReconcilePolicyServerSecret reconcile the policy server secret when necessary.
+//
+// policyServer is the Policy Server is a reference to the policy server ca secret which to be reconciled
```

Comment:
I think it this line may mean this, but I'm not sure:

```suggestion
// policyServerSecret is a reference to the policy server ca secret which to be reconciled
```